editor/plugins/sprite_2d_editor_plugin.cpp

```diff
@@ -68,7 +68,7 @@ Vector<Vector2> expand(const Vector<Vector2> &points, const Rect2i &rect, float
 	}
 	ClipperLib::ClipperOffset co;
 	co.AddPath(subj, ClipperLib::jtMiter, ClipperLib::etClosedPolygon);
-	co.Execute(solution, epsilon * PRECISION);
+	co.Execute(solution, tolerance * PRECISION);
```

Comment:
In this method, neither `epsilon` nor `tolerance` are correct. The `Execute` method parameter name is `delta`, and we are multiplying it by 10, so this is really just "a tenth of the delta given to the Clipper library".